src/lib/coda_lib/coda_lib.ml

```diff
@@ -586,6 +586,8 @@ module Make (Inputs : Inputs_intf) = struct
 
   let receipt_chain_database t = t.receipt_chain_database
 
+  let logger t = t.logger
```

Comment:
I thought we wanted to be explicit about passing the logger around so that we could make logger children later. However, this could have changed with the new design (since we pass __MODULE__ and __LINE__ anyway). @nholland94  ?

Maybe call this `top_level_logger` or `root_logger`. So we know to `extend` it as a child as necessary.

Reply:
Yah, we already get fine grain information of where errors happen via MODULE and LINE option for our logger. So, it doesn't hurt to do this.

I will rename this `top_level_logger`